django_fsm_freeze/models.py

```diff
@@ -55,17 +57,19 @@ def is_fsm_frozen(self) -> bool:
         applied.
         """
         fsm_field = self.__class__._get_fsm_field()
-        return fsm_field.value_from_object(self) in self.FROZEN_IN_STATES
+        return (
+            not self._bypass_fsm_freeze
```

Comment:
I'd like to think about this. `bypass_fsm_freeze` to me sounds more related to the freeze "checking" rather than the "frozen" status.

`is_fsm_frozen` could be in theory used by other methods (or in user's application) in the future which may not necessarily be related to bypass but just to get whether or not an object is frozen.

Reply:
My suggestion would be in PR #16 - please feel free to review and comment. Then we can see what else to further improve.